cmd/cluster-agent/api/agent/agent.go

```diff
@@ -193,15 +194,9 @@ func getWorkloadList(w http.ResponseWriter, r *http.Request, wmeta workloadmeta.
 	response := wmeta.Dump(verbose)
 	jsonDump, err := json.Marshal(response)
 	if err != nil {
-		setJSONError(w, log.Errorf("Unable to marshal workload list response: %v", err), 500)
+		httputils.SetJSONError(w, log.Errorf("Unable to marshal workload list response: %v", err), 500)
```

Comment:
Probably safer to use the `%w` to avoid some internal error being expressed with the %v format

Reply:
from the linter `github.com/DataDog/datadog-agent/pkg/util/log.Errorf does not support error-wrapping directive %w (govet)`